demo/admin/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "1.0.0",
     "private": true,
     "scripts": {
-        "admin-generator": "rimraf 'src/*/generated' && comet-admin-generator generate",
+        "admin-generator": "rimraf -g 'src/**/generated' && $npm_execpath gql:types && comet-admin-generator generate && $npm_execpath gql:types",
```

Comment:
if anyone sees a better solution i am free for suggestions.

As far as I have seen there is not easy way around this. Because 

1. admin-generator needs types before run.
2. admin-generator produces source code (without types) which should be in my opinion generated afterwards.

Reply:
If deleting all `*.generated.ts*` files and afterwards running `admin-generator` script (simulating what is happening on CI during `pnpm lint`) following error occurs, because Admin-Generator needs GQL Files to be present to work properly.

<img width="1284" alt="Screenshot 2025-03-03 at 08 15 33" src="https://github.com/user-attachments/assets/25fd224e-286e-426c-b5d5-b0800a6975a7" />